editor/plugins/org.fusesource.ide.camel.editor/src/org/fusesource/ide/camel/editor/dialogs/ConditionalBreakpointEditorDialog.java

```diff
@@ -286,6 +284,10 @@ public void widgetSelected(SelectionEvent e) {
 	    if (combo_language.getItemCount()>0) combo_language.select(0);
 	}
 	
+	private String[] getSupportedLanguages() {
+		return LanguageUtils.languageArray(node.getCamelFile().getResource().getProject());
```

Comment:
i started with the IProject as it was the most straightforward but the old code and refactored afterwards, but not this part :-)
I will do it